chrome/content/addFinger.js

```diff
@@ -0,0 +1,41 @@
+var { interfaces: Ci, utils: Cu, classes: Cc } = Components;
+
+Cu.import("resource:///modules/imServices.jsm");
+Cu.import("resource:///modules/imXPCOMUtils.jsm");
+Cu.import("chrome://otr/content/otr.js");
+
+XPCOMUtils.defineLazyGetter(this, "_", () =>
+  l10nHelper("chrome://otr/locale/addfinger.properties")
+);
+
+var args = window.arguments[0].wrappedJSObject;
+
+var otrAddFinger = {
+  onload: function() {
+    document.title = _("addfinger.title", args.screenname);
+  },
+  
+  oninput: function(e) {
+    e.value = e.value.replace(/[^0-9a-fA-F]/gi, "");
+    document.documentElement.getButton("accept").disabled = (e.value.length != 40);
+  },
+
+  add: function(e) {
+    let hex = document.getElementById("finger").value;
+    // remove exist fingerprint firstly
+    let finger = otr.hasContactFingerprint(args);
+    if (finger) {
+      finger.purge = true;
+      otr.forgetFingerprints([finger]);
+    }
```

Comment:
This isn't strictly necessary, and there're two problems:

1) If the user has already in the middle of an OTR session with the contact, `otr.forgetFingerprints()` won't forget the fingerprint. I guess that's ok.
2) `otr.hasContactFingerprint()` only returns the *first* fingerprint for the user, they may have more than one stored (think multiple devices or whatnot)

We should maybe drop the "Edit" idea, and just let you "Add". If you want to remove fingerprints, you can use the management pane for now.